packages/teleterm/webpack.renderer.dev.config.js

```diff
@@ -26,6 +26,6 @@ devCfg.devServer = {
 };
 
 devCfg.output.publicPath = '';
-devCfg.plugins = [configFactory.plugins.reactRefresh(), createHtmlPlugin()];
+devCfg.plugins.push(createHtmlPlugin());
```

Comment:
I didn't have time to investigate what causes this problem but this PR seems to break refreshing React components in Connect.

This line might not be related to that problem, I just had nowhere else to place this comment. ;)

If I open a tab in Connect and then edit its source file and save it, the dev version of Connect still shows the old version.

Reply:
Hm, I haven't tested Connect, but I will take a look. I'm not sure if this is the cause, because the react refresh plugin will already be in the array from the base config